.github/workflows/ci.yml

```
@@ -283,12 +283,14 @@ jobs:
             ~/.cargo/git/db/
             target/
           key: ${{ runner.os }}-check-doc-${{ hashFiles('**/Cargo.toml') }}
-      - uses: dtolnay/rust-toolchain@stable
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ env.NIGHTLY_TOOLCHAIN }}
```

Could you add a comment explaining why we need nightly. Something like:
```suggestion
     # Use nightly so we can use `rustdoc-scrape-examples`. See rust-lang\rust#88791
      - uses: dtolnay/rust-toolchain@master
        with:
          toolchain: ${{ env.NIGHTLY_TOOLCHAIN }}
```